components/molecules/PendingBenefitDetails.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import VerticalStepper from '../atoms/VerticalStepper'
+import en from '../../locales/en'
+import fr from '../../locales/fr'
+
+const PendingBenefitCard = (props) => {
+  const t = props.locale === 'en' ? en : fr
+
+  return (
+    <div className="grid grid-cols-1 divide-y divide-grey-500 font-display my-4">
+      <section>
+        <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-7 sm:gap-4 sm:mx-auto sm:mt-4">
+          <div
+            id="claimUpdates"
+            className="col-span-2 sm:col-span-3 lg:col-span-2 sm:w-60 ml-7 mb-6 sm:mb-3"
+          >
+            <VerticalStepper
+              benefitStatus={props.benefit.status}
+              lastUpdates={props.benefit.lastUpdates}
+              locale={props.locale}
+            />
+          </div>
+
+          <div
+            id="details"
+            className="col-span-5 sm:col-span-4 lg:col-span-5 sm:grid mr-5"
+          >
+            <div id="application_details" className="ml-5 mb-5 sm:mb-0">
+              <p className="font-bold text-xl">{t.applicationDetails}</p>
+              <p className="font-normal text-base">
+                {props.benefit.statusDescription}
+              </p>
+            </div>
+            <div id="application_description" className="ml-5 mb-5 sm:mb-0">
```

Comment:
Instead of `ml-5` per item here, just put it on the parent `<div>`

Reply:
Done